src/sentry/api/endpoints/organization_onboarding_tasks.py

```diff
@@ -62,6 +63,14 @@ def post(self, request: Request, organization) -> Response:
             values=values,
         )
 
+        if created and task_id == OnboardingTask.FIRST_PROJECT:
+            scope = sentry_sdk.get_current_scope()
+            scope.set_extra("org", organization.ids)
+            sentry_sdk.capture_message(
+                f"Onboarding task {task_id} was created unexpectedly. It should have been updated instead.",
+                level="warning",
+            )
```

Comment:
This shouldn't happen since we filter tasks by their status on the frontend, which should be either skipped or complete ([see](https://github.com/getsentry/sentry/blob/67cdeefac513ee4431fa1b9611e9f856eb592b2f/static/app/components/onboardingWizard/content.tsx#L427)), but let's see what's going on here